recipes/mpdecimal/mpdecimal.inc

```diff
@@ -0,0 +1,23 @@
+DESCRIPTION = "Package for correctly-rounded arbitrary precision decimal floating point arithmetic"
+LICENSE = "BSD-2-Clause"
+HOMEPAGE = "http://www.bytereef.org/mpdecimal/index.html"
+RECIPE_TYPES = "machine native"
+
+inherit c autotools library
+
+DEPENDS = "libm"
+SRC_URI = "http://www.bytereef.org/software/${PN}/releases/${PN}-${PV}.tar.gz"
+
+LIBRARY_NAME = "libmpdecimal"
+LIBRARY_VERSION = "2"
+
+FILES_${PN} += "${libdir}/libmpdec.so.${LIBRARY_VERSION} ${libdir}/libmpdec.so.${PV}"
+FILES_${PN}-dev += "${libdir}/libmpdec.so"
+DEPENDS_${PN} = "libc libm"
+RDEPENDS_${PN} = "libc libm"
+PROVIDES_${PN} = "libmpdec libmpdec-${LIBRARY_VERSION}"
+PROVIDES_${PN}[qa] += "allow-missing-soname:libmpdecimal-2"
+PROVIDES_${PN}[qa] += "allow-missing-soname:libmpdecimal"
```

Comment:
How is it handled when the library uses different names for header vs. library?

E.g.
```
/usr/include/mpdecimal.h
/usr/lib/libmpdec.so.2
/usr/lib/.debug/libmpdec.so.2.4.1
/usr/lib/libmpdec.so.2.4.1
```